terraso_backend/apps/collaboration/models/memberships.py

```diff
@@ -54,8 +54,16 @@ class MembershipList(BaseModel):
         default=DEFAULT_MEMERBSHIP_TYPE,
     )
 
+    def default_validation_func(self):
+        return False
```

Comment:
@paulschreiber The function using this default is the fucntion to save memberships, since the collaboration app doesn't have all the context to enforce the validation a function can be passed from the app using it to add the needed validation, if no function is passed this will be used by default